.gitmodules

```diff
@@ -73,3 +73,6 @@
 [submodule "corehq/apps/prelogin"]
 	path = corehq/apps/prelogin
 	url = https://github.com/dimagi/commcarehq-prelogin.git
+[submodule "submodules/formsplayer"]
+	path = submodules/formsplayer
+	url = git@github.com:dimagi/formsplayer.git
```

Comment:
I think for submodules we should definitely still use the https links, otherwise all our servers (and anyone who forks the repo) won't be able to update this without write access to the repo